### ISSUE TITLE
[studio-ui] Add notes in Publishing screens

### ISSUE THREAD

**Issue by alhambrav** [craftercms/craftercms#4650]
**Is your feature request related to a problem? Please describe.**
Add note in the `Publish by Commit Id` tab : 
```
Publish by commit ID(s) must be used for changes made via direct git actions against the repository or pulled from a remote repository.
```

Add note in the `Bulk Publish` tab:
```
Bulk publish should be used to publish changes made in Studio via the UI
```

**Describe the solution you'd like**
Something like this:
<img width="952" alt="Screen Shot 2021-04-14 at 9 17 05 AM" src="https://user-images.githubusercontent.com/25483966/114717179-c8d79880-9d02-11eb-9c55-7885a6caddfb.png">

**Describe alternatives you've considered**

**Additional context**
Applies to both 3.1 and 4.0

**Comment by rart** [comment]
@jvega190 That 3.1.x PR has precedence as this is for 3.1.14 (PR is missing at the moment)

**Comment by alhambrav** [comment]
Verified in 3.1

**Comment by jvega190** [comment]
@alhambrav merged in 4.0.x

**Comment by alhambrav** [comment]
Verified in 4.0
